src/Components/Layout.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Header from './Header';
+
+class Layout extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {};
+  }
+
+  render() {
+    const { children } = this.props;
+    return (
+      <div>
+        <Header />
+        <div>
+          <main>{children}</main>
```

Comment:
is there any difference so that this  `children` does not have spacing unlike the one above?

Reply:
The linting works differently, the first one `const { children } = this.props;` is just an object, and the second one `<main>{children}</main>` is jsx.

**Object spacing** https://github.com/airbnb/javascript#whitespace--in-braces
**JSX spacing** https://github.com/airbnb/javascript/tree/master/react#spacing

Reply:
The linting works differently, the first one `const { children } = this.props; `is just an object, and the second one `<main>{children}</main> `is jsx.

**Object spacing** https://github.com/airbnb/javascript#whitespace--in-braces
**JSX spacing** https://github.com/airbnb/javascript/tree/master/react#spacing